### ISSUE TITLE
there are always Error unmounting container by restart docker daemon

### ISSUE THREAD

**Issue by xiaods** [moby/moby#24744]
Hi, we use docker 1.11.2 latest release binary.

```
[root@localhost ~]# uname -r
3.10.0-327.13.1.el7.x86_64
```

```
[root@localhost ~]# docker info
Containers: 10
 Running: 10
 Paused: 0
 Stopped: 0
Images: 10
Server Version: 1.11.2
Storage Driver: devicemapper
 Pool Name: docker-253:0-393772-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop2
 Metadata file: /dev/loop3
 Data Space Used: 2.645 GB
 Data Space Total: 107.4 GB
 Data Space Available: 31.88 GB
 Metadata Space Used: 4.534 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.143 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.107-RHEL7 (2016-06-09)
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: host bridge null
Kernel Version: 3.10.0-327.13.1.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 3.703 GiB
Name: localhost.localdomain
ID: N6NR:DRXZ:Q534:WS7W:XVRU:M2MT:JOS4:VUAO:5PYH:H5UN:5CRJ:2HT2
Docker Root Dir: /var/lib/docker
Debug mode (client): false
Debug mode (server): false
Registry: https://index.docker.io/v1/

```

```
7月 18 02:53:44 localhost.localdomain systemd[1]: Starting Docker Application Container Engine...
7月 18 02:53:44 localhost.localdomain docker[2527]: time="2016-07-18T02:53:44.413724884-04:00" level=info msg="New containerd process, pid: 2532\n"
7月 18 02:53:45 localhost.localdomain docker[2527]: time="2016-07-18T02:53:45.560149703-04:00" level=warning msg="devmapper: Usage of loopback devices is strongly discouraged for production use. Please use `--
7月 18 02:53:45 localhost.localdomain docker[2527]: time="2016-07-18T02:53:45.583937781-04:00" level=warning msg="devmapper: Base device already exists and has filesystem xfs on it. User specified filesystem  
7月 18 02:53:45 localhost.localdomain docker[2527]: time="2016-07-18T02:53:45.599180355-04:00" level=info msg="[graphdriver] using prior storage driver \"devicemapper\""
7月 18 02:53:45 localhost.localdomain docker[2527]: time="2016-07-18T02:53:45.665647407-04:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
7月 18 02:53:45 localhost.localdomain docker[2527]: time="2016-07-18T02:53:45.677898759-04:00" level=info msg="Firewalld running: false"
7月 18 02:53:45 localhost.localdomain docker[2527]: time="2016-07-18T02:53:45.736071165-04:00" level=info msg="Removing stale sandbox 906c7899a92505f2b834d262fa787115e0fff8e5c3ee0995bbba387d9d12a872 (143076c19
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.066655745-04:00" level=info msg="Removing stale sandbox a6d8142ecd5963bea9e206a0aae3c6e0658bf3248cf8e99c7f94dd2a5933f843 (48ed379ba
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.389220130-04:00" level=info msg="Removing stale sandbox a9d723703f782eb23f8c892be0a5ba47ec11f53e9b3ad63420e376d0b60b5c2f (da6a8b44c
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.398318102-04:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be 
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.432227958-04:00" level=warning msg="mountpoint for pids not found"
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.432433311-04:00" level=info msg="Loading containers: start."
7月 18 02:53:46 localhost.localdomain docker[2527]: ..........time="2016-07-18T02:53:46.437399829-04:00" level=warning msg="error locating sandbox id a6d8142ecd5963bea9e206a0aae3c6e0658bf3248cf8e99c7f94dd2a593
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.437447864-04:00" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/48ed379ba19bdf600c7a
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.437473080-04:00" level=error msg="devmapper: Error unmounting device 8cf4dfb0c2d873aab801611051e124de91b906d9403f32a7b4b004d9a0a4f5
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.437480299-04:00" level=error msg="Error unmounting container 48ed379ba19bdf600c7a9ea3a0611651e28b0584c76dec36d55bef7b30410f7b: inva
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438296161-04:00" level=warning msg="error locating sandbox id 906c7899a92505f2b834d262fa787115e0fff8e5c3ee0995bbba387d9d12a872: san
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438321864-04:00" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/143076c1923b2d76ea71
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438336005-04:00" level=error msg="devmapper: Error unmounting device 823f263be0080c87f38760905f302bfbcf76b145cdd811715a51e8732caadb
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438342442-04:00" level=error msg="Error unmounting container 143076c1923b2d76ea7132981c27b6466906d138a3b93c3a8bd083843f27dd0b: inva
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438905180-04:00" level=warning msg="error locating sandbox id a9d723703f782eb23f8c892be0a5ba47ec11f53e9b3ad63420e376d0b60b5c2f: san
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438926661-04:00" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/da6a8b44cbb4d7146519
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438939700-04:00" level=error msg="devmapper: Error unmounting device ebf74d21ee71715a04a90f104921fdb149284ee8630978a636c87052390c48
7月 18 02:53:46 localhost.localdomain docker[2527]: time="2016-07-18T02:53:46.438946102-04:00" level=error msg="Error unmounting container da6a8b44cbb4d71465197c844a8b0d053720cdcde1082e0c4fad328a8f708952: inva
7月 18 02:53:47 localhost.localdomain docker[2527]: time="2016-07-18T02:53:47.592087527-04:00" level=info msg="Loading containers: done."
7月 18 02:53:47 localhost.localdomain docker[2527]: time="2016-07-18T02:53:47.592104900-04:00" level=info msg="Daemon has completed initialization"
7月 18 02:53:47 localhost.localdomain docker[2527]: time="2016-07-18T02:53:47.592115502-04:00" level=info msg="Docker daemon" commit=b9f10c9 graphdriver=devicemapper version=1.11.2
7月 18 02:53:47 localhost.localdomain docker[2527]: time="2016-07-18T02:53:47.599651322-04:00" level=info msg="API listen on /var/run/docker.sock"
7月 18 02:53:47 localhost.localdomain systemd[1]: Started Docker Application Container Engine.

```

**Comment by cpuguy83** [comment]
@xiaods Looks like your logs are truncated on the right, can you provide the full log?

**Comment by xiaods** [comment]
@cpuguy83 ok, let me try again


```
[root@localhost ~]# journalctl -u docker.service

7月 19 07:48:35 localhost.localdomain systemd[1]: Stopping Docker Application Container Engine...
7月 19 07:48:35 localhost.localdomain docker[7474]: time="2016-07-19T07:48:35-04:00" level=info msg="stopping containerd after receiving terminated"
7月 19 07:48:35 localhost.localdomain docker[7474]: time="2016-07-19T07:48:35.775762872-04:00" level=info msg="Processing signal 'terminated'"
7月 19 07:48:35 localhost.localdomain docker[7474]: time="2016-07-19T07:48:35.778601588-04:00" level=error msg="failed to receive event from containerd: rpc error: code = 13 desc = \"transport is closing\""
7月 19 07:48:38 localhost.localdomain docker[7474]: time="2016-07-19T07:48:38.782351795-04:00" level=info msg="New containerd process, pid: 5227\n"
7月 19 07:48:40 localhost.localdomain docker[7474]: time="2016-07-19T07:48:40.682641752-04:00" level=warning msg="container 24c416516a4bce1d9f23a2f3c70b361581550308eddc9b12c341ff0b33bfa450 restart canceled"
7月 19 07:48:40 localhost.localdomain docker[7474]: time="2016-07-19T07:48:40.799719444-04:00" level=warning msg="container 543c19466c70594c7c0374551c3e3503f37b56239ffb9d78ee07f504b0cbb720 restart canceled"
7月 19 07:48:41 localhost.localdomain docker[7474]: time="2016-07-19T07:48:41.197384743-04:00" level=warning msg="container 7c036e345796aa5a10a92cb1aba6dd56532e88e90932174f44af982c1e220a41 restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.644571217-04:00" level=info msg="Container 6085f5953d9b0b22677d158faac2cc053ed32cb38ef8db5a0254b6d106f9a937 failed to exit within 10 seconds of signal 15 - usin
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.644703330-04:00" level=info msg="Container 7cbe99017fd238010bdea18017217876f37587a0a8fea54faded80922fe66e17 failed to exit within 10 seconds of signal 15 - usin
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.644543313-04:00" level=info msg="Container 340800171b219348b2e731fc89e44016f38be1eef66ab7433a775292634f6f26 failed to exit within 10 seconds of signal 15 - usin
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.645415217-04:00" level=info msg="Container 94f7ef93ba565b3e120cafb2fcc7524474597a5018b552753b3b5b768e53901e failed to exit within 10 seconds of signal 15 - usin
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.645485195-04:00" level=info msg="Container 793f935661b23d17085907f38f64b960d9a60c3fcfb0723ddaa387a8430c2409 failed to exit within 10 seconds of signal 15 - usin
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.645517142-04:00" level=info msg="Container 04ff445f25fd290f7841179ee17bab23f399a319bb09c00ce9e928d954715d61 failed to exit within 10 seconds of signal 15 - usin
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.687395815-04:00" level=warning msg="container 04ff445f25fd290f7841179ee17bab23f399a319bb09c00ce9e928d954715d61 restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.747998823-04:00" level=warning msg="container 793f935661b23d17085907f38f64b960d9a60c3fcfb0723ddaa387a8430c2409 restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.751670806-04:00" level=warning msg="container 94f7ef93ba565b3e120cafb2fcc7524474597a5018b552753b3b5b768e53901e restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.761642781-04:00" level=warning msg="container 7cbe99017fd238010bdea18017217876f37587a0a8fea54faded80922fe66e17 restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.771385978-04:00" level=warning msg="container 340800171b219348b2e731fc89e44016f38be1eef66ab7433a775292634f6f26 restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.773760935-04:00" level=warning msg="container 6085f5953d9b0b22677d158faac2cc053ed32cb38ef8db5a0254b6d106f9a937 restart canceled"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50.776339031-04:00" level=error msg="Force shutdown daemon"
7月 19 07:48:50 localhost.localdomain docker[7474]: time="2016-07-19T07:48:50-04:00" level=info msg="stopping containerd after receiving terminated"
7月 19 07:48:51 localhost.localdomain systemd[1]: Stopped Docker Application Container Engine.
7月 19 07:49:02 localhost.localdomain systemd[1]: Starting Docker Application Container Engine...
7月 19 07:49:02 localhost.localdomain docker[5418]: time="2016-07-19T07:49:02.357795453-04:00" level=info msg="New containerd process, pid: 5423\n"
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.450654349-04:00" level=warning msg="devmapper: Usage of loopback devices is strongly discouraged for production use. Please use `--storage-opt dm.thinpooldev` o
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.481106961-04:00" level=warning msg="devmapper: Base device already exists and has filesystem xfs on it. User specified filesystem  will be ignored."
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.787442881-04:00" level=info msg="[graphdriver] using prior storage driver \"devicemapper\""
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.813683089-04:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.818034932-04:00" level=info msg="Firewalld running: false"
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.865679736-04:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP ad
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.899878599-04:00" level=warning msg="mountpoint for pids not found"
7月 19 07:49:03 localhost.localdomain docker[5418]: time="2016-07-19T07:49:03.900237542-04:00" level=info msg="Loading containers: start."
7月 19 07:49:03 localhost.localdomain docker[5418]: ..........time="2016-07-19T07:49:03.903294923-04:00" level=error msg="Failed to load container 884333e09e4c483b0940d9e19c3c134e1910bfc33cceee3351c8fabcbd930936: open /var/lib/docker/cont
7月 19 07:49:05 localhost.localdomain docker[5418]: .
7月 19 07:49:05 localhost.localdomain docker[5418]: time="2016-07-19T07:49:05.239317947-04:00" level=info msg="Loading containers: done."
7月 19 07:49:05 localhost.localdomain docker[5418]: time="2016-07-19T07:49:05.239334663-04:00" level=info msg="Daemon has completed initialization"
7月 19 07:49:05 localhost.localdomain docker[5418]: time="2016-07-19T07:49:05.239349077-04:00" level=info msg="Docker daemon" commit=b9f10c9 graphdriver=devicemapper version=1.11.2
7月 19 07:49:05 localhost.localdomain docker[5418]: time="2016-07-19T07:49:05.246740654-04:00" level=info msg="API listen on /var/run/docker.sock"
7月 19 07:49:05 localhost.localdomain systemd[1]: Started Docker Application Container Engine.
7月 19 07:49:11 localhost.localdomain docker[5418]: time="2016-07-19T07:49:11-04:00" level=error msg="containerd: notify OOM events" error="write /sys/fs/cgroup/memory/docker/85596c898377185898642a9bdd215846bded7ec498cbe52b2a493ee408a0514
7月 19 07:49:56 localhost.localdomain docker[5418]: time="2016-07-19T07:49:56-04:00" level=error msg="containerd: notify OOM events" error="write /sys/fs/cgroup/memory/docker/85596c898377185898642a9bdd215846bded7ec498cbe52b2a493ee408a0514
7月 19 07:50:48 localhost.localdomain docker[5418]: time="2016-07-19T07:50:48-04:00" level=error msg="containerd: notify OOM events" error="write /sys/fs/cgroup/memory/docker/85596c898377185898642a9bdd215846bded7ec498cbe52b2a493ee408a0514
7月 19 07:55:55 localhost.localdomain docker[5418]: time="2016-07-19T07:55:55-04:00" level=error msg="containerd: notify OOM events" error="write /sys/fs/cgroup/memory/docker/85596c898377185898642a9bdd215846bded7ec498cbe52b2a493ee408a0514
7月 19 08:02:45 localhost.localdomain docker[5418]: time="2016-07-19T08:02:45-04:00" level=error msg="containerd: notify OOM events" error="write /sys/fs/cgroup/memory/docker/85596c898377185898642a9bdd215846bded7ec498cbe52b2a493ee408a0514




```

**Comment by xiaods** [comment]
```
$ vagrant version
Installed Version: 1.8.4
Latest Version: 1.8.5
```

**Comment by bsousaa** [comment]
It looks like this issue went stale. Let me close it for now. Feel free to comment and we can re-open if we have more info.
